.travis.yml

```diff
@@ -23,12 +25,14 @@ script:
   - test -z "$(go vet $(find ./arm/* -type d -print) | tee /dev/stderr)"
   - test -z "$(golint ./arm/... | tee /dev/stderr)"
   - go build ./arm/examples/...
-  - go test -v ./management/...
+  - if [[ $TRAVIS_GO_VERSION != 1.6* ]]; then
```

Comment:
Ah, I think we're alluding to the same thing. For folks who install the SDK with the command I mention above, it'll will try to build each package, including the management ones. They'll get the same error that Travis was running into.

We can solve that by documenting and have folks expect that err, or we could add build tags to all of the packages in management. :) I imagine a little scripting could make that happen in no time flat.